src/components/AddToDoSection/AddToDoSection.tsx

```diff
@@ -9,9 +9,10 @@ const labels = {
 
 type AddToDoSectionType = {
   groupId?: string;
+  onClickAddTask?: () => void;
```

Comment:
onAddTaskButtonClick
Why you pass additional onAddTaskButtonClick handler but not for addTaskGroup?
What is the purpose of it here?

Reply:
the purpose was to attach active class on the taskList to change it to visible. According to new concept of hidden addToDoSection it will be deleted.